Zappy_IA_Report/Rapport_IA.md

```
@@ -0,0 +1,246 @@
+# Mise à jour de pip pour éviter les conflits
+pip install --upgrade pip
+```
+
+### 3.4 Organisation du Code
+
+Le code TensorFlow a été organisé dans le dossier **NN (Neural Network)**, suivant une approche structurée pour faciliter la maintenance et l'évolution du projet de réseaux de neurones.
+
+### 3.5 Ressources d'Apprentissage Utilisées
+
+L'apprentissage s'est appuyé sur plusieurs ressources complémentaires :
+- **Playlist complète** : Série de tutoriels couvrant les aspects fondamentaux de TensorFlow
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)